prod-refractr.yml

```diff
@@ -1256,3 +1252,12 @@ refracts:
 - github.com/mozilla/contribute.json:
   - contributejson.org
   - www.contributejson.org
+
+# bug 1255190
+- www.openwebgames.com/: openwebgames.com
+  status: 302
+
```

Comment:
we can get rid of this

Reply:
Thanks! Removed in [`8140f20` (#311)](https://github.com/mozilla-it/refractr/pull/311/commits/8140f20b6cb47b70a6d5ce15eae839e52f1b4626)